requirements_dev.txt

```diff
@@ -0,0 +1,13 @@
+# Semi-required for pyopencl
+mako
+
+# Semi-required for meshpy source install, avoids warning and wait
+pybind11
```

Comment:
We've removed `meshpy`, so this can go.

Reply:
meshpy comes back as a dependency from grudge.

Reply:
Where? I [don't see it](https://github.com/inducer/grudge/blob/2b7122d08e3a2ed22a59cb06754776eadf2573d8/setup.py#L46-L59).

Reply:
Here: https://github.com/inducer/grudge/blob/2b7122d08e3a2ed22a59cb06754776eadf2573d8/requirements.txt